test/gurka/test_disable_hierarchy.cc

```diff
@@ -0,0 +1,54 @@
+#include "gurka.h"
```

Comment:
this isnt a gurka test, it doesnt create osm data. we have another test for costing options: https://github.com/valhalla/valhalla/blob/master/test/parse_request.cc

i would suggest moving it there or to an other existing test. our build times are already long. the more tiny test files we have to compile the worse it gets (also gets harder to find stuff). long term we should organize the testing more but in the meantime we should avoid making our lives harder

Reply:
Sure! Should I test the warning in gurka? It needs to create an ascii map for location and distance information, I think.

Reply:
well the location comes from the request, no actual osm data or routing tiles are needed. we have tons of tests in loki_service that test all the errors that come from this sort of "bad request" checking. since this is only a warning, which is not testing in loki service yet, its a little bit more difficult to test because the warning actually doesnt stop the request from trying to route. still though you can do a more traditional unit test style. you can simply instantiate a loki worker and just call worker::route/matrix etc and see how it modified the request object. it either should or shouldnt have the warning added depending on your test case.

Reply:
Hi Kevin, is there a way to avoid/bypass routing in `loki_worker::route` action? I tried initiating a loki worker and calling `route` action; by adding cheating code in `worker::route` I found the warning is functioning as expected. But as you said, it's just a warning so routing is not stopped. Errors of building tiles keep popping up, and they will cover up the warning.
Maybe the best way to test is to build an ascii map in gurka?

Reply:
Thanks for the feedback! You are right about the exception. I'm working on it.